base/src/main/java/com/smartdevicelink/managers/file/DeleteFileOperation.java

```diff
@@ -66,6 +70,13 @@ private void start() {
         if (getState() == Task.CANCELED) {
             return;
         }
+        if (!mutableRemoteFileNames.contains(fileName) && completionListener != null) {
```

Comment:
```suggestion
        if (!mutableRemoteFileNames.contains(fileName)) {
            if (completionListener != null) {
                String errorMessage = "File to delete is no longer on the head unit, aborting operation";
                // Returning BaseFileManager.SPACE_AVAILABLE_MAX_VALUE for bytesAvaialble as a placeHolder, it will not get updated in BaseFileManager as long as success returned is false.
                completionListener.onComplete(false, BaseFileManager.SPACE_AVAILABLE_MAX_VALUE, mutableRemoteFileNames, errorMessage);
            }
            onFinished();
            return;
        }
```

Do we still want to call onFinished and return if the completionListener is null?